sys/net/dsm/dsm.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_dsm
+ * @{
+ *
+ * @file
+ * @brief       DTLS Session Management module implementation
+ *
+ * @author      János Brodbeck <janos.brodbeck@ml-pa.com>
+ *
+ * @}
+ */
+
+#include "net/dsm.h"
+#include "mutex.h"
+#include "net/sock/util.h"
+#include "xtimer.h"
+
+#define ENABLE_DEBUG 1
```

Comment:
```suggestion
#define ENABLE_DEBUG 0
```

Reply:
Urgh... I thought about it just before pushing and then forgot about it